### PR TITLE
Remove private_endpoint from Proxy

### PR DESCRIPTION
**Pull request by josemigallas**
**What this PR does / why we need it**:

Since APIAP is now the default, there's no need for this rolling update

**Which issue(s) this PR fixes** 

TBD

### CODE REVIEW

File: app/models/proxy.rb [hunk 1]
```diff
@@ -685,27 +685,4 @@ def self.extract_domain(url)
...
-      return if attribute_value.blank?
-
-      begin
-        uri = URI.parse(attribute_value)
-        value = URI::Generic.new(uri.scheme, uri.userinfo, uri.host, uri.port, uri.registry, uri.path, uri.opaque, uri.query, uri.fragment).to_s
-        @model[attribute] = value
-      rescue URI::InvalidURIError
-        @model.errors.add(attribute, 'Invalid domain')
-      end
-    end
```
Comment (Martouta) [inline app/models/proxy.rb:705]: we still use this... inside this same class

File: app/models/proxy.rb [hunk 2]
```diff
@@ -685,27 +685,4 @@ def self.extract_domain(url)
...
-  end
-
-  def generate_port(proxy_attribute)
-    PortGenerator.new(self).call(proxy_attribute)
-  end
```
Comment (Martouta) [inline app/models/proxy.rb:710]: we still use this... inside this same class

**Review by Martouta — CHANGES_REQUESTED** [review]
we cannot remove the port generator because we use it for the proxy model

File: app/models/backend_api.rb [hunk 3]
```diff
@@ -38,9 +40,6 @@ class BackendApi < ApplicationRecord
 
   alias_attribute :api_backend, :private_endpoint
```
Comment (Martouta) [inline app/models/backend_api.rb:41]: but now the private_endpoint is removed... so, what about this?

**Review by Martouta — DISMISSED** [review]

**Comment by Martouta** [comment]
U will have to fix the tests tho 👍🏼

**Review by Martouta — CHANGES_REQUESTED** [review]
TODO:
1. Fix the tests of CircleCI.
2. Extract to other PRs the unrelated stuff to what the title/description of the PR says

**Comment by hallelujah** [comment]
Sorry, it seems your work did not get enough attention from us.
Thank you for making this project better.

We are going to close it as it was stale for too long.

**Pull request closed without merging**